src/Utils/MultiStepInput.ts

```diff
@@ -0,0 +1,196 @@
+/*
```

Comment:
I'm trying to add this file on #547. If you need this file before merging my PR, you can add it on your PR first. :)